src/main/java/com/ss/training/utopia/counter/service/CancellationService.java

```diff
@@ -29,9 +33,10 @@
 	}
 
 	@Transactional
-	public Boolean cancelBooking(Long travelerId, Long flightId) {
+	public Boolean cancelBooking(Long travelerId, Long flightId) throws StripeException {
 		Flight flight;
 		Booking booking;
+		Stripe.apiKey = "sk_test_51GwErbJwa8c7tq3Odc3WXzypPn0OPpPAd6O5gjvRBBEb15K77CX8D2XSGyyXYgbpNJ0tW52TNRY8ox0o8iKgTkqj00v7B6meHs";
```

Comment:
Hey even thought these are test keys, we should never put any api keys on github. even if you remove it later it will always be on github now unless we remove these commits.